modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc

```diff
@@ -150,6 +150,27 @@ subjects:
 On {kubernetes}, `che` ServiceAccount must have a cluster-wide `list` and `get` `namespaces` permissions as well as an `admin` role in target namespace.
 ====
 
+[#disabling-creating-namespace-with-server-sa]
```

Comment:
WDYT about negating this? I mean write somewhere on this page that namespace is created by che-server service account and rewrite this paragraph into "How to create namespaces with user account"?

Reply:
You right. Originally this article was build when it was disabled by default. After it was changed to true I don't see a value in it anymore.